Website with navigation

amklass11 opened the pull request:
@amklass11  and @Fatima-hub333  implemented
1.ui for the library app
2.linter errors fixed

Fatima-hub333 committed:
Create Navigation bar & give styling

Fatima-hub333 committed:
Update scripts.js add functionalities of navigation bar and other sections of page

amklass11 committed:
debuggin errors html

amklass11 committed:
fix js errors

amklass11 committed:
fixed linter errros

amklass11 committed:
final linter fix

hiromataba commented on index.html:
```diff
@@ -9,25 +9,62 @@
+  <div id="books"></div>
+</div>
+
+<form class="addbook d-flex justify-content-center">
+  <h2 class="mt-5 text-center">Add a New Book</h2>
+  <div class="d-flex justify-content-center mt-5">
+    <label for="titulo"></label>
+    <input class="w-50 p-1" type="text" id="titulo" placeholder="Title of Book">
+  </div>
+  <div class="mt-3 d-flex justify-content-center">
+    <label for="autor"></label>
+    <input  class="w-50 p-1" type="text" id="autor" placeholder="Author of Book">
+  </div>
+  <div class="d-flex justify-content-center">
+  <button id="btn">Add</button>
+</div>
+</form>
```
- [x] Please make your form match the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) now it looks like this :point_down: 

![image](https://user-images.githubusercontent.com/75126481/165755748-d9587bf2-c293-4391-848b-9bbe14988c4c.png)

- Original design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/165755839-36d27c91-7717-4884-bdad-ebb2da86adef.png)

hiromataba commented on index.html:
```diff
@@ -9,25 +9,62 @@
+  </nav>
+</header>
+
+<main>
```
- [x] When any of the links are clicked, the main area changes to display only one of the three content sections but for now your main area is showing all three sections, you can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods

![image](https://user-images.githubusercontent.com/75126481/165754621-e4e0e8c9-da82-476f-ad2c-f82df6981961.png)

hiromataba reviewed (changes requested):
Hi  @amklass11  and @Fatima-hub333  :wave: :wave: 

![almost-there](https://user-images.githubusercontent.com/75126481/160092588-b3f37e8e-a1d9-4b14-84fc-7ec1fed7b3e5.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Fatima-hub333 committed:
fix ui issues in form section

Fatima-hub333 committed:
Change styling for UI & fix issues

- I changed the bootstrap styling into simple CSS as it creates issues in the JS functionality of adding an event listener.

Lordkaito reviewed (approved):
Hi @amklass11, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
